### ISSUE TITLE
[BUG] Hours Played not shown with privacy options, no warning/notification from Depressurizer

### ISSUE THREAD

**Issue by mahaniok** [Depressurizer/Depressurizer#135]
**Describe the bug**
In Depressurizer UI, I see that every game has 0 in Hours Played column

**To Reproduce**
Start Depressurizer and load your Steam profile.

Reload does the same

**Expected behavior**
Should load and show actual hours played.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version 4.9.0

**Comment by mvegter** [comment]
Do you have a public profile and have you set the privacy option for your profile to Public and Game details also on public?

**Comment by mahaniok** [comment]
Yes. My profile is fully public.

**Comment by mahaniok** [comment]
I found a checkbox "Always keep my total playtime private even if users can
see my game details." that was checked. After I unchecked it, Depressurizer
started seeing my non-zero Hours Played.

So it's not necessarily a bug, but it's unexpected. I would assume that
Depressurizer, since it's a local app with access to all local files and
registry, would ignore the privacy settings of Steam. If it's not true, may
I suggest that it would be great to make it clear in the UI. Thank you!

Best,
Ihar Mahaniok
